runtime/bcutil/ClassFileOracle.hpp

```diff
@@ -1066,6 +1066,7 @@ class RecordComponentIterator
 #if JAVA_SPEC_VERSION >= 16
 		SCOPED_ANNOTATION,
 #endif /* JAVA_SPEC_VERSION >= 16*/
+		NOT_CHECKPOINT_SAFE_ANNOTATION,
```

Comment:
```suggestion
#if defined(J9VM_OPT_CRIU_SUPPORT)
		NOT_CHECKPOINT_SAFE_ANNOTATION,
#endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
```

Reply:
I'll resolve this in favour of the new PR.